sklearn/metrics/ranking.py

```diff
@@ -226,6 +227,9 @@ def roc_auc_score(y_true, y_score, average="macro", sample_weight=None):
     sample_weight : array-like of shape = [n_samples], optional
         Sample weights.
 
+    pos_label : int or str, default=None
+        The label of the positive class. Only make sense for binary y_true.
```

Comment:
default=None means nothing here. Don't say it. You can say "optional" then describe the default below. I think the default should be to take the greater of two class labels, or 1 if only one class is present. State that for multilabel, it is fixed to 1.